src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,7 +10,7 @@
  * Represents a Person in the address book.
  * Guarantees: details are present and not null, field values are validated.
  */
-public class Person implements ReadOnlyPerson {
+public class Person implements ReadOnlyPerson, Comparable<Person> {
```

Comment:
Nice to see you implementing polymorphism for a natural ordering of the Person class.